AISKU/Init.ts

```diff
@@ -92,5 +87,5 @@ try {
     //     // core.track(telemetryItem);
     // }
 } catch (e) {
-    _InternalLogging.warnToConsole('Failed to initialize AppInsights JS SDK: ' + e.message);
+    // _InternalLogging.warnToConsole('Failed to initialize AppInsights JS SDK: ' + e.message);
```

Comment:
Add comment here we need to find proper home to warn to console when SDK initialization fails.